aws/resource_aws_vpc_endpoint.go

```diff
@@ -367,72 +434,6 @@ func vpcEndpointWaitUntilDeleted(conn *ec2.EC2, vpceId string, timeout time.Dura
 	return err
 }
 
-func vpcEndpointAttributes(d *schema.ResourceData, vpce *ec2.VpcEndpoint, conn *ec2.EC2) error {
```

Comment:
Replaced by direct setting of attributes:
* https://github.com/terraform-providers/terraform-provider-aws/pull/8396/files#r277149439
* https://github.com/terraform-providers/terraform-provider-aws/pull/8396/files#r277149449